src/features/governance/components/ProposalApprovalsTable.tsx

```diff
@@ -0,0 +1,72 @@
+import { SpinnerWithLabel } from 'src/components/animation/Spinner';
+import { Identicon } from 'src/components/icons/Identicon';
+import { Collapse } from 'src/components/menus/Collapse';
+import { ShortAddress } from 'src/components/text/ShortAddress';
+import { useProposalApprovers } from 'src/features/governance/hooks/useProposalApprovers';
+import { Address } from 'viem';
+
+export function ProposalApprovalsTable({ proposalId }: { proposalId: number }) {
+  const {
+    isLoading,
+    confirmedBy: confirmations,
+    requiredConfirmationsCount: required,
+  } = useProposalApprovers(proposalId);
+  const fraction = isLoading ? '? / 3' : `${confirmations?.length || 0} / ${required}`;
```

Comment:
why 3 ?

Reply:
3 is the required number (currently) so i was optimistically using that